docs/lib/auth/fragments/js/advanced.md

```diff
@@ -1,3 +1,15 @@
+## Back Doors
```

Comment:
Do we use this language elsewhere? We typically refer to this kind of facility as an "escape hatch" so I'm curious about why you chose this nomenclature.

Reply:
To @jpignata point here, we need to make sure that we are following a similar language or else we need to have legal review this. can we discuss this @harrysolovay and where this is pull request came from?